docs/content/contributing/community/developer-meetings.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Developer Meetings"
+date: 2018-09-28T10:40:50-07:00
+weight: 2
+
+---
+
+We hold our weekly developer meetings every Thursday at 11am US Pacific Time (UTC-7). They start with a quick update on the project from the Maintainers, and then we discuss things on the agenda.
```

Comment:
I think we should consider leaving the timezone out as we would have to change it with daylight savings twice a year.

Reply:
Agreed

Reply:
nit: Change "Maintainers" to "maintainers"

Reply:
Thanks, I've made a change to remove UTC-7 as suggested. 👍